help/en/html/doc/Technical/getgitcode.shtml

```diff
@@ -159,15 +159,9 @@
 
       <p>After the download is complete, there are menu items under
       the application "Repository" menu to access the code
-      directory:<br>
-      <a href="images/GitDtRepoSettingsMenu.png"><img src=
-      "images/GitDtRepoSettingsMenu.png" alt=
```

Comment:
> As you're not using this image any longer, can it be deleted or updated?

I took out the link because the image wasn't there for it to link to.  Does it perhaps have a different name?  I.e. I don't understand the question.